cmd/buildtools/utils.go

```diff
@@ -255,6 +255,38 @@ func GetLatestGitHubTag(ctx context.Context, owner, repo string) (string, error)
 	return tags[0].GetName(), nil
 }
 
+func GetBestGitHubTag(ctx context.Context, owner, repo string, constrants *semver.Constraints) (string, error) {
```

Comment:
What does "Best" mean? :)

Reply:
baybe just `GetGitHubTag`?

Reply:
more descriptive would be something like GetGreatestGitHubTag